prost-build/src/code_generator.rs

```diff
@@ -38,6 +38,12 @@ pub struct CodeGenerator<'a> {
     buf: &'a mut String,
 }
 
+fn _push_indent(buf: &mut String, depth: u8) {
```

Comment:
Not following why this has a `_`?

Reply:
Fixed

My C++ brain made me bug, I needed a local private free function which name would not conflict with the class method...